ecorise-react-app/src/utils/RewardApi.js

```diff
@@ -0,0 +1,8 @@
+
+export async function fetchJSON(url, options = {}) {
```

Comment:
If this file is for getting rewards, why not just add the base url here? Why is this file name starting with an uppercase? Did we not state that the util files should be named as `fetchRewards`?
Why is the function called `fetchJSON`?